[MTA] [RN] Release notes for 8.2.0 (#370)
migtools/mta-documentation: docs/topics/release-notes-topics/ref_fixed-issues-8-2-0.adoc

:_newdoc-version: 2.18.7
:_template-generated: 2026-06-12
:_mod-docs-content-type: REFERENCE

[id="fixed-issues-8-2-0_{context}"]
= Fixed issues

[role="_abstract"]

> Review issues that have been fixed in {ProductFirstRef} version 8.2.0.

Reviewer: @dymurray Could you review the fixed issues in 8.2, please?

:_newdoc-version: 2.18.7
:_template-generated: 2026-06-12
:_mod-docs-content-type: REFERENCE

[id="fixed-issues-8-2-0_{context}"]
= Fixed issues

[role="_abstract"]

> Review issues that have been fixed in {ProductFirstRef} version 8.2.0.

Reviewer 2: I think we should use the same formatting for MTA we used in New features and enhancements, WDYT?

```suggestion
Review issues that have been fixed in {ProductFullName} version 8.2.0.
```

Reviewer: Yup, that's right. Thanks, fixed it :+1:

> == {ProductShortName} command-line interface 

Reviewer 2: I like the idea of splitting fixed issues per component. As a next step, I'd consider splitting all RN types for consistent experience. WDYT?

Reviewer: Yes :) I think we should start categorizing by components from the next release.

{ProductShortName} analysis generates a record when you use an HTTP or HTTPS proxy::

Before this update, {ProductShortName} did not generate an analysis log when you used the `--http-proxy` or `--https-proxy` option in an {ProductShortName} CLI analysis. As a consequence, you could not verify if {ProductShortName} routed the analysis through the proxy. With this fix, you can verify proxy use because {ProductShortName} analysis records the use of a proxy.
+
link:https://redhat.atlassian.net/browse/MTA-6553[MTA-6553]

== {ProductShortName} web console

// Please confirm if the following bug must be documented?

> {ProductShortName} discovers Maven application that uses self-aigned certificate::

Reviewer 2: ```suggestion
{ProductShortName} discovers Maven application that uses self-signed certificates::
```

In addition to the typo, I wonder if we can rephrase "{ProductShortName} discovers Maven application" as it sounds a bit anthropomorphic. How about something like this instead?

```suggestion
{ProductShortName} processes Maven application that uses self-signed certificates::
```
or

```suggestion
{ProductShortName} supports Maven application that uses self-signed certificates::
```

Reviewer: I understand this but it's a little tricky...in MTA, discovery is a process that MTA performs. So, is it ok if I keep "discovers" here?

Reviewer 2: Ah sure, in this case let's keep the original wording.

> 

Reviewer 3: @Pkylas007 Please remove any comments (//) you no longer need.

> Before this update, after you enabled the option to *Consume insecure artifact repositories* in the {ProductShortName} web console, {ProductShortName} could not discover Maven applications that use self-signed certificates to connect to {ProductShortName}. As a consequence, you could not generate deployment assets for the affected applications. With this fix, {ProductShortName} connects with Maven application repositories that use a self-signed certificate and completes application discovery.   

Reviewer 2: Anthropomorphism

```suggestion
Before this update, after you enabled the option to *Consume insecure artifact repositories* in the {ProductShortName} web console, {ProductShortName} failed to process Maven applications that use self-signed certificates to connect to {ProductShortName}. As a consequence, you could not generate deployment assets for the affected applications. With this fix, {ProductShortName} connects with Maven application repositories that use a self-signed certificate and completes application discovery.   
```

Reviewer: I've modified it here, thanks!

+
link:https://redhat.atlassian.net/browse/MTA-7361[MTA-7361]

// Please confirm if this a fixed issue?
{ProductShortName} web console validates YAML custom rules::

Before this update, the {ProductShortName} web console did not validate the custom YAML rules you upload for an analysis or in custom targets. As a consequence, if a rule contained invalid keys or syntax errors, you did not see an error message. With this fix, {ProductShortName} improves user experience by validating the custom rules you upload in the web console.
+
link:https://redhat.atlassian.net/browse/MTA-6834[MTA-6834]

{ProductShortName} triggers violation for open source dependencies in Gradle applications::

> Before this update, {ProductShortName} did not trigger violations for open source dependency libraries when analyzing Gradle applications. As a consequence, a `source+dependency` analysis generated less incidents in the static report. With this fix, {ProductShortName} includes violations on open source dependencies in the Gradle analysis results.

Reviewer 2: As the noun “incidents” is countable.

```suggestion
Before this update, {ProductShortName} did not trigger violations for open source dependency libraries when analyzing Gradle applications. As a consequence, a `source+dependency` analysis generated fewer incidents in the static report. With this fix, {ProductShortName} includes violations on open source dependencies in the Gradle analysis results.
```

Reviewer: Done, thank you!
